src/main/kotlin/com/terraformation/backend/tracking/api/ObservationsController.kt

```diff
@@ -665,6 +689,32 @@ data class ListAssignedPlotsResponsePayload(
     val plots: List<AssignedPlotPayload>,
 ) : SuccessResponsePayload
 
+data class ReplaceObservationPlotRequestPayload(
+    val duration: ReplacementDuration,
+    val justification: String,
+)
+
+data class ReplaceObservationPlotResponsePayload(
+    @Schema(
+        description =
+            "IDs of monitoring plots that were added to the observation. Empty if no plots were " +
+                "added.")
+    val addedMonitoringPlotIds: Set<MonitoringPlotId>,
+    @Schema(
+        description =
+            "IDs of monitoring plots that were removed from the observation. Will usually " +
+                "include the requested plot ID, but may be empty if the replacement request " +
+                "couldn't be satisfied.")
+    val removedMonitoringPlotIds: Set<MonitoringPlotId>,
+) : SuccessResponsePayload {
+  constructor(
+      result: ReplacementResult
+  ) : this(
+      addedMonitoringPlotIds = result.addedMonitoringPlotIds,
+      removedMonitoringPlotIds = result.removedMonitoringPlotIds,
```

Comment:
Figma shows a success message like this.
```
Plot <plot-name-x1> has been reassigned to <plot-name-x4>.
Plot <plot-name-x7> has been reassigned to <plot-name-x9>.
```
Should the result be a map of removedId to addedId (or null if not replaced)?

Reply:
Please ignore my comment. I will work with Chudo on fixing the message for the permanent cluster case.

Reply:
Another option is to list the removed/replaced in some specific order, direction of plot perhaps.

Reply:
Copying my comments from Figma for completeness:

* There isn't a 1-to-1 mapping per se in the permanent cluster case, though for presentation purposes you could use the direction.
* Another case to consider: the request succeeds but there is no replacement plot. We do that with permanent plots in some circumstances: we just remove them from the observation but don't add another plot to make up for them.